SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -433,8 +433,10 @@ bool FSpatialGDKEditorToolbarModule::ValidateGeneratedLaunchConfig() const
 		return false;
 	}
 
-	const ULevelEditorPlaySettings* LevelEditorPlaySettings = GetDefault<ULevelEditorPlaySettings>();
-	if (!SpatialGDKRuntimeSettings->bEnableHandover && LevelEditorPlaySettings->GetTotalPIEServerWorkerCount() > 1)
+	if (!SpatialGDKRuntimeSettings->bEnableHandover && SpatialGDKEditorSettings->LaunchConfigDesc.ServerWorkers.ContainsByPredicate([](const FWorkerTypeLaunchSection& Section)
```

Comment:
Could there be any problem if `LevelEditorPlaySettings->GetTotalPIEServerWorkerCount() > 1` doesn't match this predicate?

Reply:
GetTotalPIEServerWorkerCount() is a slightly stale value to use here, better to use the load balancer to values directly.

Reply:
But it's still safe to use `GetTotalPIEServerWorkerCount()` in `EditorWorkerController::InitWorkers`?

Reply:
Yes, because that's using the count to determine how many servers to launch from within the editor. You can connect external server-workers to your local deployment. This check is if your load-balancer is setup for zoned workers and handover is disabled, which would always be incorrect.

Reply:
I'll update the message dialog to more accurate reflect this.